docs/25_developer.md

```diff
@@ -184,32 +187,43 @@ the build system.*
 
 In order to use the Postgres database in development, one must:
 
-1. Install PostgreSQL (brew install postgres, apt-get install
-   postgres, or whatever is appropriate) and start it running.
-2. Create a database called "corla", and grant all privileges on it to
-   a user called "corla" with password "corla".
-3. Initialize the "corla" database with test administrator data.
-For example, to accomplish the above on OS X using brew, one issues
-the following commands:
+1. Install PostgreSQL (`brew install postgres` on OS X, `apt-get
+   install postgresql` on many Linux distributions, or whatever is
+   appropriate) and start it running.
+2. Create a database called "`corla`", and grant all privileges on it
+   to a user called "`corla`" with password "`corla`".
+3. Initialize the "`corla`" database with test administrator data.
+   For example, to accomplish the above on OS X using Homebrew, one
+   issues the following commands:
 ```
 brew install postgres
 createuser -P corla
 createdb -O corla corla
 ```
-4. Run the server (to create all the database tables).
-5. Load test authentication credentials into the database, by executing
-the SQL in `corla-test-credentials.sql` (found in the `test` directory
-of the repository). This can be done with the following command on OS X:
-```
-psql -U corla -d corla -a -f corla-test-credentials.psql
-```
+On Linux, one would replace the first command with something akin to
+`sudo apt-get install postgresql`.
 
 That's it. If the database is there the server will use it and will,
 at this stage, create all its tables and such automatically.
+4. Run the server (to create all the database tables). Recall that
+   this is accomplished by either running the server in Eclipse using
+   the Run button or running it from a command line using a command
+   akin to `java -jar colorado_rla-VERSION-shaded.jar`.
+5. Load test authentication credentials into the database, by
+   executing the SQL in `corla-test-credentials.sql` (found in the
```

Comment:
s/.sql/.psql (this one is important because otherwise they won't find it; though it's correct below)